metabase/metabase
# Add global instance context for Snowplow events

Epic: https://github.com/metabase/metabase/issues/18291

Adds `instance` information to every Snowplow event:
- Instance ID
- Instance version
- Token features

Also fixes the issue with missing user IDs in initial `pageview` by waiting for loading user data.

How to test:
- Install Snowplow Debug Extension
- Change `isProduction` to true, run Metabase, and open home page
- Verify that `pageview` event is fired with the data

I've also tested that the data is available in our Metabase instance, meaning it was processed by Snowplow correctly.
